org-formation/600-access/organization-tasks.yaml

```diff
@@ -6,11 +6,76 @@ NlpSandboxServiceAccount:
   StackName: NlpSandboxServiceAccount
   MaxConcurrentStacks: 10
   Parameters:
-    ManagedPolicyArns:
-      - arn:aws:iam::aws:policy/AmazonEC2FullAccess
-      - arn:aws:iam::aws:policy/AWSCloudFormationFullAccess
-      - arn:aws:iam::aws:policy/AmazonS3FullAccess
+    PolicyDocument: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "ec2",
+                "Effect": "Allow",
+                "Action": [
+                    "ec2:AuthorizeSecurityGroupEgress",
+                    "ec2:AuthorizeSecurityGroupIngress",
+                    "ec2:CreateTags",
+                    "ec2:CreateSecurityGroup",
+                    "ec2:DescribeInstances",
+                    "ec2:DescribeSecurityGroups",
+                    "ec2:DescribeSubnets",
+                    "ec2:DeleteSecurityGroup",
+                    "ec2:DeleteTags",
+                    "ec2:RevokeSecurityGroupEgress",
+                    "ec2:RunInstances",
+                    "ec2:TerminateInstances",
+                    "elasticloadbalancing:AddTags",
+                    "elasticloadbalancing:DescribeTargetGroups",
+                    "elasticloadbalancing:DescribeTargetHealth",
+                    "elasticloadbalancing:DescribeLoadBalancers",
+                    "elasticloadbalancing:DescribeListeners",
+                    "elasticloadbalancing:DeleteTargetGroup",
+                    "elasticloadbalancing:DeleteLoadBalancer",
+                    "elasticloadbalancing:RemoveTags",
+                    "elasticloadbalancing:CreateListener",
+                    "elasticloadbalancing:CreateTargetGroup",
+                    "elasticloadbalancing:CreateLoadBalancer",
+                    "elasticloadbalancing:RegisterTargets"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Sid": "elb",
+                "Effect": "Allow",
+                "Action": "elasticloadbalancing:*",
+                "Resource": "*"
+            },
+            {
+                "Sid": "s3",
+                "Effect": "Allow",
+                "Action": [
+                    "s3:PutObject"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::bootstrap-awss3cloudformationbucket-z6hnifi9i3p/*"
```

Comment:
Scicomp user accesses a different bucket I think.

Reply:
it's `arn:aws:s3:::bootstrap-awss3cloudformationbucket-114n2ojlbvj21` on scicomp.  you can probably just do `arn:aws:s3:::bootstrap-awss3cloudformationbucket-*`